### PR TITLE
add theme handling for mapbox

### LINKED ISSUE #91: 2J - Theme handling for map
- [x] pouvoir changer de thème de carte en temps réel
- [x] proposer plusieurs choix de thème
- [x] bouton pour changer de thème
- [x] quand l'utilisateur lance l'application, la map doit avoir le dernier thème choisi
- [x] mettre la bonne valeur par défaut dans le spinner

### PR DESCRIPTION
**Pull request by larroyo3**
Closes #91 
--

Issues terminé.

Axes d'amélioration  (je peux pas faire tant que j''ai pas la maquette :cry: ): 
- mettre en place les vrais thème chois dans la maquette 
- mettre le vrai bouton changement de thème (bon endroit et bonne forme)

J'ai pas mi de screen vu que j'ai rien fait de graphique, j'ai juste mi un vieux bouton dans la page settings pour pouvoir tester

Faire attention au moment de merge les 2 PR active risque important de conflit vu que j'ai pas pu merge l'autre sur dev avant de faire celle la

### CODE REVIEW

File: app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
 import com.bouboulecorp.thurii.R
 import com.mapbox.maps.MapView
 import com.mapbox.maps.Style
 
+
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt:12]: 🚫 **[ktlint]** <no-consecutive-blank-lines> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Needless blank line(s)

File: app/src/main/java/com/bouboulecorp/thurii/MainActivity.kt [hunk 2]
```diff
@@ -1,4 +1,5 @@
 package com.bouboulecorp.thurii
+import android.content.Context
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/MainActivity.kt:2]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/MainActivity.kt:2]: 🚫 **[ktlint]** <no-unused-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 3]
```diff
@@ -1,8 +1,13 @@
 package com.bouboulecorp.thurii.Settings
+import android.content.Context
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:2]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 4]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_settings)
+
+        // access the items of the list
+        val theme = resources.getStringArray(R.array.theme)
+
+        // access the spinner
+        val spinner = findViewById<Spinner>(R.id.spinner)
+        if (spinner != null) {
+            val adapter = ArrayAdapter(this,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:27]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline after "("

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:27]: 🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 5]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                android.R.layout.simple_spinner_item, theme)
+            spinner.adapter = adapter
+
+            val sharedPreference =  getSharedPreferences("USER_VARIABLES",Context.MODE_PRIVATE)
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:31]: 🚫 **[ktlint]** <comma-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ","

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 6]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+            when (sharedPreference.getString("mapbox_style","mapbox://styles/mapbox/outdoors-v11")) {
+                "mapbox://styles/mapbox/dark-v10" ->  spinner.setSelection(1) //dark theme
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:33]: 🚫 **[ktlint]** <comment-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing space after //

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 7]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                "mapbox://styles/mapbox/satellite-streets-v11" -> spinner.setSelection(2) //satellite theme
+                else -> {
+                    spinner.setSelection(0)
+                }
+            }
+
+            spinner.onItemSelectedListener = object :
+                AdapterView.OnItemSelectedListener {
+                override fun onItemSelected(parent: AdapterView<*>,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:42]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline after "("

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:42]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 8]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                                            view: View, position: Int, id: Long) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:43]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:43]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 9]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                    Toast.makeText(this@SettingsActivity,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:44]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline after "("

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 10]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                        getString(R.string.selected_item) + " " +
+                                "" + theme[position], Toast.LENGTH_SHORT).show()
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:46]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (32) (should be 28)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:46]: 🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:46]: 🚫 **[ktlint]** <wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:46]: 🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing newline before ")"

**Review by Neyross — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>ktlint</summary>

[app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt|51 col 97|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/818a72e5aa835f0698c019a1f81bd1bb20eaa52b/app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt#L51) Missing space after //
[app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt|52 col 62|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/818a72e5aa835f0698c019a1f81bd1bb20eaa52b/app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt#L52) Missing spacing after ","
[app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt|52 col 110|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/818a72e5aa835f0698c019a1f81bd1bb20eaa52b/app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt#L52) Missing space after //
[app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt|54 col 61|](http://github.com/Bouboule-Corp/thurii-mobile-kotlin/blob/818a72e5aa835f0698c019a1f81bd1bb20eaa52b/app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt#L54) Missing spacing after ","
</details>

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 11]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+
+                    val editor = sharedPreference.edit()
+
+                    when (position) {
+                        1 -> editor.putString("mapbox_style","mapbox://styles/mapbox/dark-v10") //dark theme
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:51]: 🚫 **[ktlint]** <comment-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing space after //

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 12]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                        2 -> editor.putString("mapbox_style","mapbox://styles/mapbox/satellite-streets-v11") //satellite theme
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:52]: 🚫 **[ktlint]** <comma-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ","

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 13]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                        else -> {
+                            editor.putString("mapbox_style","mapbox://styles/mapbox/outdoors-v11")
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:54]: 🚫 **[ktlint]** <comma-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ","

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 14]
```diff
@@ -12,6 +17,52 @@ class SettingsActivity : AppCompatActivity() {
...
+                                            view: View, position: Int, id: Long) {
+                    Toast.makeText(this@SettingsActivity,
+                        getString(R.string.selected_item) + " " +
+                                "" + theme[position], Toast.LENGTH_SHORT).show()
+
+                    val editor = sharedPreference.edit()
+
+                    when (position) {
+                        1 -> editor.putString("mapbox_style","mapbox://styles/mapbox/dark-v10") //dark theme
+                        2 -> editor.putString("mapbox_style","mapbox://styles/mapbox/satellite-streets-v11") // satellite theme
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:52]: 🚫 **[ktlint]** <comma-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ","

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 15]
```diff
@@ -12,6 +17,55 @@ class SettingsActivity : AppCompatActivity() {
...
+
+        // access the items of the list
+        val theme = resources.getStringArray(R.array.theme)
+
+        // access the spinner
+        val spinner = findViewById<Spinner>(R.id.spinner)
+        if (spinner != null) {
+            val adapter = ArrayAdapter(
+                this,
+                android.R.layout.simple_spinner_item,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:29]: 🚫 **[ktlint]** <no-trailing-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing space(s)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:29]: 🚫 **[ktlint]** <no-multi-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary long whitespace

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 16]
```diff
@@ -12,6 +17,55 @@ class SettingsActivity : AppCompatActivity() {
...
+        val spinner = findViewById<Spinner>(R.id.spinner)
+        if (spinner != null) {
+            val adapter = ArrayAdapter(
+                this,
+                android.R.layout.simple_spinner_item, 
+                theme
+            )
+            spinner.adapter = adapter
+
+            val sharedPreference = getSharedPreferences("USER_VARIABLES",Context.MODE_PRIVATE)
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:34]: 🚫 **[ktlint]** <comma-spacing> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing spacing after ","

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 17]
```diff
@@ -12,6 +17,60 @@ class SettingsActivity : AppCompatActivity() {
...
+                "mapbox://styles/mapbox/satellite-streets-v11" -> spinner.setSelection(2) // satellite theme
+                else -> {
+                    spinner.setSelection(0)
+                }
+            }
+
+            spinner.onItemSelectedListener = object :
+                AdapterView.OnItemSelectedListener {
+                override fun onItemSelected(
+                                        parent: AdapterView<*>,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:46]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (40) (should be 20)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 18]
```diff
@@ -12,6 +17,60 @@ class SettingsActivity : AppCompatActivity() {
...
+                                        view: View, position: Int, id: Long
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:47]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (40) (should be 20)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:47]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:47]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 19]
```diff
@@ -12,6 +17,60 @@ class SettingsActivity : AppCompatActivity() {
...
+                                        ) {
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:48]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (40) (should be 16)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 20]
```diff
@@ -12,6 +17,60 @@ class SettingsActivity : AppCompatActivity() {
...
+                    Toast.makeText(
+                        this@SettingsActivity,
+                        getString(R.string.selected_item) + " " +
+                        "" + theme[position],
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:52]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (24) (should be 28)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 21]
```diff
@@ -12,6 +17,61 @@ class SettingsActivity : AppCompatActivity() {
...
+                else -> {
+                    spinner.setSelection(0)
+                }
+            }
+
+            spinner.onItemSelectedListener = object :
+                AdapterView.OnItemSelectedListener {
+                override fun onItemSelected(
+                    parent: AdapterView<*>,
+                    view: View, position: Int,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:47]: 🚫 **[ktlint]** <parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should be on a separate line (unless all parameters can fit a single line)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:47]: 🚫 **[ktlint]** <no-trailing-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing space(s)

Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:47]: 🚫 **[ktlint]** <no-multi-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unnecessary long whitespace

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 22]
```diff
@@ -12,6 +17,61 @@ class SettingsActivity : AppCompatActivity() {
...
+
+            spinner.onItemSelectedListener = object :
+                AdapterView.OnItemSelectedListener {
+                override fun onItemSelected(
+                    parent: AdapterView<*>,
+                    view: View, position: Int, 
+                    id: Long
+                ) {
+                    Toast.makeText(
+                            this@SettingsActivity,
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:51]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (28) (should be 24)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 23]
```diff
@@ -12,6 +17,61 @@ class SettingsActivity : AppCompatActivity() {
...
+                            getString(R.string.selected_item) + " " +
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:52]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (28) (should be 24)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 24]
```diff
@@ -12,6 +17,61 @@ class SettingsActivity : AppCompatActivity() {
...
+                            "" + theme[position],
+                            Toast.LENGTH_SHORT
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:54]: 🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (28) (should be 24)

**Comment by ArthurBoucard** [comment]
@larroyo3 si la pr est faite et que la branch sert plus a rien il faus delete la branch

**Comment by larroyo3** [comment]
Gestion des thèmes pour la map fonctionnel.

Il ne restera plus qu'à mettre le bouton au bonne endroit avec la nouvelle maquette

File: app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt [hunk 25]
```diff
@@ -1,13 +1,15 @@
 package com.bouboulecorp.thurii.HomePage.HomePageFragments
+
+import android.content.Context
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end

File: app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt [hunk 26]
```diff
@@ -41,6 +43,16 @@ class MapFragment : Fragment() {
...
+
+        val preferences = this.activity!!
+            .getSharedPreferences("USER_VARIABLES", Context.MODE_PRIVATE)
+        val mapboxStyle = preferences.getString("mapbox_style", "mapbox://styles/mapbox/outdoors-v11")
+
+        mapView = view.findViewById(R.id.mapView)
+        if (mapboxStyle != null) {
+            mapView?.getMapboxMap()?.loadStyleUri(mapboxStyle)
+        }
+
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt:55]: 🚫 **[ktlint]** <no-trailing-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing space(s)

File: app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt [hunk 27]
```diff
@@ -94,6 +106,7 @@ class MapFragment : Fragment() {
                 }
             }
         )
+
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/HomePage/HomePageFragments/MapFragment.kt:109]: 🚫 **[ktlint]** <no-trailing-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing space(s)

File: app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt [hunk 28]
```diff
@@ -17,6 +21,62 @@ class SettingsActivity : AppCompatActivity() {
...
+
+                    editor.commit()
+                }
+
+                override fun onNothingSelected(parent: AdapterView<*>) {
+                    // write code to perform some action
+                }
+            }
+        }
+
```
Comment (Neyross) [inline app/src/main/java/com/bouboulecorp/thurii/Settings/SettingsActivity.kt:79]: 🚫 **[ktlint]** <no-trailing-spaces> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing space(s)

**Pull request closed without merging**